### ISSUE TITLE
Renderer info doesn't differentiate between Impeller and Skia

### ISSUE THREAD

**Issue by ueman** [getsentry/sentry-dart#1719]
### Platform

Flutter Mobile

### Obfuscation

Disabled

### Debug Info

Disabled

### Doctor

Irrelevant 

### Version

7.12.0

### Steps to Reproduce

1. cause error
2. track error
3. see Skia as renderer even when Impeller is used

### Expected Result

Sentry can identify whether Skia or Impeller is used

### Actual Result

It's always Skia

### Are you willing to submit a PR?

None

**Comment by ueman** [comment]
As seen in https://github.com/getsentry/sentry-dart/blob/212d6c7b8070c9999404017f21fdf7afdd840f7b/flutter/lib/src/renderer/io_renderer.dart#L4 Skia is hardcoded on dart:io platforms.

I guess it's probably to detect Impeller vs Skia based on the Info.plist flag on iOS. Maybe Android has something similar.

**Comment by buenaflor** [comment]
Android should have a flag in `AndroidManifest.xml`. Is it possible to access flags from those during runtime?

**Comment by ueman** [comment]
> Android should have a flag in `AndroidManifest.xml`. Is it possible to access flags from those during runtime?

Yeah, but not from Flutter, just from the native parts. I would just remove it from the Flutter side for now (no info is better than wrong info) and then add the correct info in a later step.
